frontend/components/atoms/AOutlinedButton.tsx

```diff
@@ -0,0 +1,17 @@
+import { FC } from 'react';
+import { Button } from '@material-ui/core';
+
+interface AOutlinedButtonProps {
+  onClick: React.MouseEventHandler<HTMLButtonElement>;
+  buttonText: string;
+}
+
+const AOutlinedButton: FC<AOutlinedButtonProps> = ({ onClick, buttonText, ...other }) => {
+  return (
+    <Button color="primary" variant="outlined" onClick={onClick}>
+      {buttonText}
```

Comment:
```suggestion
    <Button color="primary" variant="outlined" onClick={onClick} {...other}>
      {buttonText}
```

Do remember to pass `other` to the custom prop, so we can still utilise the props from material UI's button.